tests/lib/certificateSign.js

```diff
@@ -44,6 +44,34 @@ define([
           );
       });
 
+      test('#with metricsContext metadata', function () {
+        return accountHelper.newVerifiedAccount()
+          .then(function (account) {
+            var publicKey = {
+              algorithm: 'RS',
+              n: '4759385967235610503571494339196749614544606692567785790953934768202714280652973091341316862993582789079872007974809511698859885077002492642203267408776123',
+              e: '65537'
+            };
+            var duration = 86400000;
+
+            return respond(
+              client.certificateSign(account.signIn.sessionToken, publicKey, duration, {
+                metricsContext: {
+                  flowId: '0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef',
+                  flowBeginTime: Date.now(),
+                  forbiddenProperty: 666
```

Comment:
Fwiw, I've pushed some variation in the tests, so that they're at least exercising some different variations of the `metricsContext` properties. Hopefully that's a bit better than what we had before.